src/routes/index/_rewards.svelte

```diff
@@ -17,9 +17,9 @@
 					<h3 class="mb-3 text-xl lg:text-[2rem] font-semibold">4 contributions or more</h3>
 
 					<p class="leading-relaxed">
-						Limited edition Appwrite Hacktoberfest T-shirt<br />
-						Sticker pack<br />
-						Socks
+						Limited edition Appwrite Hacktoberfest T-shirt,<br />
+						Sticker pack,<br />
+						Socks.
```

Comment:
Lists in traditional grammatically correct English use punctuation but I have removed this change as per request

Reply:
I think we should take a second look at the readability here. Maybe it didn't come across this way to readers. 

Maybe it's because of how long the first line is :P WDYT
<img width="375" alt="Screen Shot 2022-10-04 at 9 54 21 AM" src="https://user-images.githubusercontent.com/29069505/193838147-438697f0-0d23-4007-8ce5-bac6113db918.png">

Reply:
Maybe just `Limited Edition T-Shirt` because Appwrite and Hacktoberfest are implied in context?

Reply:
> Maybe just `Limited Edition T-Shirt` because Appwrite and Hacktoberfest are implied in context?

I mean now we're in a whole different world of changes, I wouldn't know as that I didn't write this part?